bundles/org.openhab.binding.oppo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -118,6 +127,10 @@
 		<item-type>Number:Time</item-type>
 		<label>Time Display (S)</label>
 		<description>The Playback Time Elapsed/Remaining in Seconds</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Duration</tag>
+		</tags>
 		<state readOnly="true" pattern="%d %unit%"/>
```

Comment:
Sorry for offtopic question, but seeing this, I'm wondering: Do you think it would make sense to change the pattern to e.g. `%1$tT`? It seems more natural for a player to display elapsed/remaining time as e.g. 3:17 rather than 197 s.

Reply:
Yes it should probably be done here instead of externally via javascript as indicated in the current README example. I will change this for all my current PRs.